README.md

```diff
@@ -1,61 +1,122 @@
 
 # YYJ Tech Code of Conduct
 
+YYJ Tech strives to be as equitible and inclusive as possible. This code of conduct sets the expectations and guidelines for all members on how to conduct themselves to create a community that is as welcoming and inclusive as possible, as well as the consequences for unacceptable behavior.
+
+However, we are all humans, and we know that there are times that we will not get things right. 
+
+Help us out by sharing things you are concerned about here. You can find more information about how to do that in the [Reporting Guidelines](https://github.com/yyjtech/code-of-conduct/blob/master/reporting_guidelines.md).
+
+
 ## 1\. Purpose
 
 A primary goal of YYJ Tech is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
 
-This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
-
 We invite all those who participate in the YYJ Tech community to help us create safe and positive experiences for everyone.
 
 ## 2\. Open Tech Citizenship
 
-A supplemental goal of this Code of Conduct is to increase open tech citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community.
+A supplemental goal of this Code of Conduct is to increase open tech citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their impacts on our community.
 
 Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.
 
-If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to celebrate and support that.
```

Comment:
Oooh, nice, I *really* like this change to the wording.

Reply:
Plus 1